src/absil/ilreflect.fs

```diff
@@ -1388,10 +1388,10 @@ let convMethodImplFlags mdef =
 //----------------------------------------------------------------------------
   
 let rec buildMethodPass2 cenv tref (typB:TypeBuilder) emEnv (mdef : ILMethodDef) =
-   // remaining REVIEW:
-   // SecurityDecls: Permissions;
-   // IsUnmanagedExport: bool; (* -- The method is exported to unmanaged code using COM interop. *)
-   // IsMustRun: bool; (* Whidbey feature: SafeHandle finalizer must be run *)
+    // remaining REVIEW:
```

Comment:
It was written by the original developer listing the parts of the ILMethodDef data structure that have not yet been consumed by this function.  The parts are only used on mscorlib or in mixed-mode native/managed assemblies, or COM interop code, none of which is generated by ILReflect.  So the comment can be removed.